sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSource.scala

```diff
@@ -602,7 +602,7 @@ object DataSource extends Logging {
                   provider1.startsWith("org.apache.spark.sql.hive.orc")) {
                   throw new AnalysisException(
                     "Hive-based ORC data source must be used with Hive support enabled. " +
-                    "Please use native ORC data source instead")
+                    "Please use native ORC data source instead by `SET spark.sql.orc.impl=native`")
```

Comment:
Wait .. @dongjoon-hyun can we just say like `by setting 'spark.sql.orc.impl' configuration to 'native'`?

Reply:
Sure!